internal/k8s/converter.go

```diff
@@ -93,8 +93,8 @@ func NewUnstructuredConverter() (*UnstructuredConverter, error) {
 	}
 
 	// Setup converter to understand custom CRD types
-	projectcontour.AddKnownTypes(uc.scheme)
-	ingressroutev1.AddKnownTypes(uc.scheme)
+	projectcontour.AddToScheme(uc.scheme) // nolint:errcheck
```

Comment:
Should probably just add the error checking here, we do it below for everything else that uses `AddToScheme`

Reply:
'kk, will do 👍

Reply:
Fixed.